modules/meeting/spec/support/pages/meetings/show.rb

```diff
@@ -445,12 +445,15 @@ def clear_backlog
     def edit_agenda_item(item, save: true, &)
       select_action item, "Edit"
       expect_item_edit_form(item)
+      reference_value = meeting_reference_value
       page.within("#meeting-agenda-items-form-component-#{item.id}") do
         yield
         if save
           click_save_and_wait_for_agenda_item_update
         end
       end
+
+      wait_for_reference_changed(reference_value)
```

Comment:
I had no idea that there was something stable to be waited for when updating an agenda item.

I think this `wait_for_reference_changed(reference_value)` should be surrounded with `if save`. 

I tried and it does fix some tests, but not all of them.
It looks like updating backlog items does not update the `data-reference-value` value.